### ISSUE TITLE
Container run failure: Failed to create a directory "/output/.classCache" for the shared class cache

### ISSUE THREAD

**Issue by smhgit** [IBM/FHIR#814]
Fail running container.

```
eyalco-mac:fhir-install eyalco$ docker run -it -p 9080:9080 -p 9443:9443 --name fhir-server --rm fhir-server
cp: cannot create regular file '/config/configDropins/defaults/keystore.xml': No such file or directory

JVMSHRC559E Failed to create a directory "/output/.classCache" for the shared class cache
JVMSHRC840E Failed to start up the shared cache.
JVMSHRC686I Failed to startup shared class cache. Continue without using it as -Xshareclasses:nonfatal is specified
Launching fhir-server (WebSphere Application Server 20.0.0.3/wlp-1.0.38.cl200320200305-1433) on IBM J9 VM, version 8.0.6.6 - pxa6480sr6fp6-20200303_01(SR6 FP6) (en_US)
[AUDIT   ] CWWKE0001I: The server fhir-server has been launched.
[AUDIT   ] CWWKE0100I: This product is licensed for development, and limited production use. The full license terms can be viewed here: https://public.dhe.ibm.com/ibmdl/export/pub/software/websphere/wasdev/license/base_ilan/ilan/20.0.0.3/lafiles/en.html
[AUDIT   ] CWWKG0028A: Processing included configuration resource: /opt/ibm/wlp/usr/servers/fhir-server/batchDs.xml
[AUDIT   ] CWWKZ0058I: Monitoring dropins for applications.
[AUDIT   ] CWWKS4104A: LTPA keys created in 2.226 seconds. LTPA key file: /opt/ibm/wlp/usr/servers/fhir-server/resources/security/ltpa.keys
[AUDIT   ] CWWKT0016I: Web application available (default_host): https://1066535f642d:9443/openapi/ui/
[AUDIT   ] CWWKT0016I: Web application available (default_host): https://1066535f642d:9443/jwt/
[AUDIT   ] CWWKT0016I: Web application available (default_host): https://1066535f642d:9443/ibm/api/
[AUDIT   ] CWWKT0016I: Web application available (default_host): https://1066535f642d:9443/oidc/
[AUDIT   ] CWWKT0016I: Web application available (default_host): https://1066535f642d:9443/openapi/
[AUDIT   ] CWWKT0016I: Web application available (default_host): https://1066535f642d:9443/oauth2/
[WARNING ] SRVE9967W: The manifest class path pdq.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-server-webapp.war/WEB-INF/lib/jcc-11.5.0.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_cs.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-server-webapp.war/WEB-INF/lib/derbytools-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_de_DE.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-server-webapp.war/WEB-INF/lib/derbytools-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_es.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-server-webapp.war/WEB-INF/lib/derbytools-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_fr.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-server-webapp.war/WEB-INF/lib/derbytools-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_hu.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-server-webapp.war/WEB-INF/lib/derbytools-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_it.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-server-webapp.war/WEB-INF/lib/derbytools-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_ja_JP.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-server-webapp.war/WEB-INF/lib/derbytools-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_ko_KR.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-server-webapp.war/WEB-INF/lib/derbytools-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_pl.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-server-webapp.war/WEB-INF/lib/derbytools-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_pt_BR.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-server-webapp.war/WEB-INF/lib/derbytools-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_ru.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-server-webapp.war/WEB-INF/lib/derbytools-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_zh_CN.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-server-webapp.war/WEB-INF/lib/derbytools-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_zh_TW.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-server-webapp.war/WEB-INF/lib/derbytools-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_cs.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-bulkimportexport-webapp.war/WEB-INF/lib/derby-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_de_DE.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-bulkimportexport-webapp.war/WEB-INF/lib/derby-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_es.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-bulkimportexport-webapp.war/WEB-INF/lib/derby-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_fr.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-bulkimportexport-webapp.war/WEB-INF/lib/derby-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_hu.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-bulkimportexport-webapp.war/WEB-INF/lib/derby-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_it.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-bulkimportexport-webapp.war/WEB-INF/lib/derby-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_ja_JP.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-bulkimportexport-webapp.war/WEB-INF/lib/derby-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_ko_KR.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-bulkimportexport-webapp.war/WEB-INF/lib/derby-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_pl.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-bulkimportexport-webapp.war/WEB-INF/lib/derby-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_pt_BR.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-bulkimportexport-webapp.war/WEB-INF/lib/derby-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_ru.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-bulkimportexport-webapp.war/WEB-INF/lib/derby-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_zh_CN.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-bulkimportexport-webapp.war/WEB-INF/lib/derby-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_zh_TW.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-bulkimportexport-webapp.war/WEB-INF/lib/derby-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_cs.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-bulkimportexport-webapp.war/WEB-INF/lib/derby-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_de_DE.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-bulkimportexport-webapp.war/WEB-INF/lib/derby-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_es.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-bulkimportexport-webapp.war/WEB-INF/lib/derby-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_fr.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-bulkimportexport-webapp.war/WEB-INF/lib/derby-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_hu.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-bulkimportexport-webapp.war/WEB-INF/lib/derby-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_it.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-bulkimportexport-webapp.war/WEB-INF/lib/derby-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_ja_JP.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-bulkimportexport-webapp.war/WEB-INF/lib/derby-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_ko_KR.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-bulkimportexport-webapp.war/WEB-INF/lib/derby-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_pl.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-bulkimportexport-webapp.war/WEB-INF/lib/derby-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_pt_BR.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-bulkimportexport-webapp.war/WEB-INF/lib/derby-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_ru.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-bulkimportexport-webapp.war/WEB-INF/lib/derby-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_zh_CN.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-bulkimportexport-webapp.war/WEB-INF/lib/derby-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_zh_TW.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-bulkimportexport-webapp.war/WEB-INF/lib/derby-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path pdq.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-server-webapp.war/WEB-INF/lib/jcc-11.5.0.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_cs.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-server-webapp.war/WEB-INF/lib/derbytools-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_de_DE.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-server-webapp.war/WEB-INF/lib/derbytools-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_es.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-server-webapp.war/WEB-INF/lib/derbytools-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_fr.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-server-webapp.war/WEB-INF/lib/derbytools-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_hu.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-server-webapp.war/WEB-INF/lib/derbytools-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_it.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-server-webapp.war/WEB-INF/lib/derbytools-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_ja_JP.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-server-webapp.war/WEB-INF/lib/derbytools-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_ko_KR.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-server-webapp.war/WEB-INF/lib/derbytools-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_pl.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-server-webapp.war/WEB-INF/lib/derbytools-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_pt_BR.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-server-webapp.war/WEB-INF/lib/derbytools-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_ru.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-server-webapp.war/WEB-INF/lib/derbytools-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_zh_CN.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-server-webapp.war/WEB-INF/lib/derbytools-10.14.2.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLocale_zh_TW.jar can not be found in jar file file:/opt/ibm/wlp/usr/servers/fhir-server/apps/expanded/fhir-server-webapp.war/WEB-INF/lib/derbytools-10.14.2.0.jar or its parent.
[AUDIT   ] CWWKT0016I: Web application available (default_host): https://1066535f642d:9443/fhir-openapi/
[AUDIT   ] CWWKZ0001I: Application fhir-openapi started in 7.225 seconds.
[AUDIT   ] CWWKT0016I: Web application available (default_host): https://1066535f642d:9443/fhir-bulkimportexport-webapp/
[AUDIT   ] CWWKZ0001I: Application fhir-bulkimportexport-webapp started in 9.595 seconds.
[AUDIT   ] CWWKT0016I: Web application available (default_host): https://1066535f642d:9443/fhir-server/api/v4/
[WARNING ] [ONCE] Create tablespace not supported in Derby
[WARNING ] [ONCE] Derby does not support CREATE VARIABLE for: SV_TENANT_ID
[WARNING ] [ONCE] Create procedure not supported in Derby
[WARNING ] [ONCE] Derby does support not multi-tenancy: PARAMETER_NAMES
[WARNING ] [ONCE] Derby does not support CREATE PERMISSION for: PARAMETER_NAMES_TENANT
[WARNING ] [ONCE] Derby does not support ROW ACCESS CONTROL for table: PARAMETER_NAMES
[AUDIT   ] CWWKF0012I: The server installed the following features: [appSecurity-2.0, batch-1.0, batchManagement-1.0, distributedMap-1.0, el-3.0, federatedRegistry-1.0, jaxrs-2.1, jaxrsClient-2.1, jdbc-4.1, jndi-1.0, json-1.0, jsonp-1.1, jsp-2.3, ldapRegistry-3.0, mpConfig-1.2, mpOpenAPI-1.0, oauth-2.0, openidConnectServer-1.0, servlet-4.0, ssl-1.0, transportSecurity-1.0, websocket-1.1].
[AUDIT   ] CWWKF0011I: The fhir-server server is ready to run a smarter planet. The fhir-server server started in 42.437 seconds.
[AUDIT   ] CWWKZ0022W: Application fhir-server-webapp has not started in 29.967 seconds.
[AUDIT   ] CWWKZ0001I: Application fhir-server-webapp started in 142.304 seconds.
```

**Comment by prb112** [comment]
thanks @smhgit

**Comment by prb112** [comment]
Updated the Dockerfile and now in Review/QA

**Comment by prb112** [comment]
This is indirectly addressed in #904

**Comment by prb112** [comment]
Evidence
```
Digest: sha256:c8f1be5144ca4b28caa6332fe3d65fcae4e6a45d76ef60d131491990a7d68961
Status: Downloaded newer image for ibmcom/ibm-fhir-server:latest

Launching fhir-server (Open Liberty 20.0.0.3/wlp-1.0.38.cl200320200305-1433) on Eclipse OpenJ9 VM, version 1.8.0_242-b08 (en_US)
[AUDIT   ] CWWKE0001I: The server fhir-server has been launched.
[AUDIT   ] CWWKG0093A: Processing configuration drop-ins resource: /opt/ol/wlp/usr/servers/fhir-server/configDropins/defaults/keystore.xml
[AUDIT   ] CWWKG0028A: Processing included configuration resource: /opt/ol/wlp/usr/servers/fhir-server/batchDs.xml
[AUDIT   ] CWWKG0093A: Processing configuration drop-ins resource: /opt/ol/wlp/usr/servers/fhir-server/configDropins/overrides/trustDefault.xml
[AUDIT   ] CWWKG0102I: Found conflicting settings for defaultKeyStore instance of keyStore configuration.
  Property password has conflicting values:
    Secure value is set in file:/opt/ol/wlp/usr/servers/fhir-server/configDropins/defaults/keystore.xml.
    Secure value is set in file:/opt/ol/wlp/usr/servers/fhir-server/server.xml.
  Property password will be set to the value defined in file:/opt/ol/wlp/usr/servers/fhir-server/server.xml.

[AUDIT   ] CWWKZ0058I: Monitoring dropins for applications.
[AUDIT   ] CWWKS4104A: LTPA keys created in 4.059 seconds. LTPA key file: /opt/ol/wlp/usr/servers/fhir-server/resources/security/ltpa.keys
[AUDIT   ] CWWKT0016I: Web application available (default_host): https://4f96a9b8d23e:9443/openapi/ui/
[AUDIT   ] CWWKT0016I: Web application available (default_host): https://4f96a9b8d23e:9443/openapi/
[AUDIT   ] CWWKT0016I: Web application available (default_host): https://4f96a9b8d23e:9443/ibm/api/
[AUDIT   ] CWWKT0016I: Web application available (default_host): https://4f96a9b8d23e:9443/jwt/
[AUDIT   ] CWWKT0016I: Web application available (default_host): https://4f96a9b8d23e:9443/oidc/
[WARNING ] SRVE9967W: The manifest class path pdq.jar can not be found in jar file file:/opt/ol/wlp/usr/servers/fhir-server/apps/expanded/fhir-server-webapp.war/WEB-INF/lib/jcc-11.5.0.0.jar or its parent.
[WARNING ] SRVE9967W: The manifest class path derbyLoc
```
